bundler/spec/commands/newgem_spec.rb

```diff
@@ -569,7 +569,13 @@ def create_temporary_dir(dir)
     it "sets a minimum ruby version" do
       bundle "gem #{gem_name}"
 
-      expect(generated_gemspec.required_ruby_version).to eq(Gem::Requirement.new(Gem.ruby_version < Gem::Version.new("2.4.a") ? ">= 2.3.0" : ">= 2.4.0"))
+      required_ruby_version = if Gem.ruby_version < Gem::Version.new("2.4.a") then ">= 2.3.0"
+      elsif Gem.ruby_version < Gem::Version.new("2.5.a") then ">= 2.4.0"
+      elsif Gem.ruby_version < Gem::Version.new("2.6.a") then ">= 2.5.0"
+      else
+        ">= 2.6.0"
+      end
+      expect(generated_gemspec.required_ruby_version).to eq(Gem::Requirement.new(required_ruby_version))
```

Comment:
At this point it feels like we are just reimplementing the same logic we are testing inside the test itself. I think I'd change to spec to just check that a `required_rub_version` property is generated with a lower bound (starts with >=).

Reply:
Thanks, I might misunderstand your intention, but I've rewritten that spec to simply check `>=`.

Reply:
If you observe the previous logic here, and the logic you added inside the `required_ruby_version` method in `bundler/lib/bundler/cli/gem.rb`, it's exactly the same. That means that in order to test some logic, we are using exactly the same logic that we want to get tested. The defeats the point of having a test.

It'd be like using a spec like

`expect(sum(1, 1)).to eq(1 + 1)`

to test the method

```
def sum(a, b)
  a +  b
end
```

Reply:
@deivid-rodriguez Could you check the current code? It checks using `>=` only.

Reply:
Yes, it looks good, thank you!